code/obj/flock/structure/rift.dm

```diff
@@ -7,6 +7,7 @@
 	density = FALSE
 	name = "glowing portal thingymabob"
 	desc = "Oh god is that a fucking light grenade?!"
+	flock_desc = "The rift through which your Flock will enter this world."
```

Comment:
```suggestion
	flock_desc = "The rift through which your Flock will begin its journey."
```
Or similar? Sounds cool, but maybe feels a bit out of theme, idk

Reply:
🤷‍♀️